### DISCUSSION TITLE
Debug No group identified in quad search

### DISCUSSION THREAD

**Discussion by timellemeet** [ERGO-Code/HiGHS#1629] [Q&A] [answered]
Hey, somtimes I get a `kNotSet` model status and I identified now this happens when we get `DualChuzC: No group identified in quad search so return error`. Do you have some guidance as to handle this? Do you expect this is something in my problem formulation? Should I maybe retry with a different strategy (I do need to get duals out for column generation).

![image](https://github.com/ERGO-Code/HiGHS/assets/6830281/ba0f1fd2-6cdd-40e3-8304-33f8e861b015)

**Comment by jajhall** [comment]
This can be a consequence of models with large values.

I'm travelling, so will look at this properly later today

**Comment by jajhall** [comment, marked as answer]
> Hey, somtimes I get a `kNotSet` model status and I identified now this happens when we get `DualChuzC: No group identified in quad search so return error`. Do you have some guidance as to handle this? Do you expect this is something in my problem formulation? Should I maybe retry with a different strategy (I do need to get duals out for column generation).
> 
> ![image](https://private-user-images.githubusercontent.com/6830281/306639462-ba0f1fd2-6cdd-40e3-8304-33f8e861b015.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg1NTU3NDYsIm5iZiI6MTcwODU1NTQ0NiwicGF0aCI6Ii82ODMwMjgxLzMwNjYzOTQ2Mi1iYTBmMWZkMi02Y2RkLTQwZTMtODMwNC0zM2Y4ZTg2MWIwMTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMjFUMjI0NDA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWYyYmNmM2I0ZmEyMThlY2VlNDAxYmViZjc0N2NkMDg0NmVmNWZkYzk0MzZhYmI3MDY1NDk3ZTA3ZDIzZTNkMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.c9s2D9jKC7qgZc--x6NDjfWWjLiVdwLpVpf9yGZa4Qs)

This happens when the dual values are so large that a (different!) perturbation strategy fails - because, when its small values are added to the dual values, they aren't changed.
